operator/internal/handlers/internal/storage/secrets.go

```diff
@@ -56,11 +56,15 @@ func extractAzureConfigSecret(s *corev1.Secret) (*storage.AzureStorageConfig, er
 		return nil, kverrors.New("missing secret field", "field", "account_key")
 	}
 
+	// Extract and validate optional fields
+	endpointSuffix := s.Data["endpoint_suffix"]
```

Comment:
I'm a bit confused by the comment, we are only extracting the field correct?

Reply:
It's just a hint to make the section more human readable. I imagine that we extra more optional fields if requested.